templates/components/cart/content.html

```diff
@@ -1,192 +1,135 @@
+                                class="js-form-input js-form-input-increment-total js-cart-item-qty-input c-cart__item-qty-input"
+                                id="qty-{{id}}"
+                                name="qty-{{id}}"
+                                type="tel"
+                                value="{{quantity}}"
+                                data-quantity-min="{{min_purchase_quantity}}"
+                                data-quantity-max="{{max_purchase_quantity}}"
+                                data-quantity-min-error="{{lang 'products.quantity_min' quantity=min_purchase_quantity}}"
+                                data-quantity-max-error="{{lang 'products.quantity_max' quantity=max_purchase_quantity}}"
+                                min="1"
+                                pattern="[0-9]*"
                                 data-cart-itemid="{{id}}"
-                                data-action="inc"
-                            >
-                                <span class="u-hidden-visually">
-                                    {{lang 'products.quantity_increase' name=name}}
-                                </span>
-                                <svg>
-                                    <use href="#icon-keyboard-arrow-up" />
-                                </svg>
-                            </button>
-                        {{/if}}
+                                data-action="manualQtyChange"
+                                aria-label="{{name}}"
+                                aria-live="polite"{{#unless can_modify}} disabled{{/unless}}>
+                            {{#if can_modify}}
+                                <button 
+                                    class="js-cart-update c-cart__item-qty-btn"
+                                    data-cart-itemid="{{id}}"
+                                    data-action="inc"
+                                >
+                                    <span class="u-hidden-visually">
+                                        {{lang 'products.quantity_increase' name=name}}
+                                    </span>
+                                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+                                        <path d="M8 12H16" stroke="#373D3F" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                                        <path d="M12 16V8" stroke="#373D3F" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                                        <path d="M9 22H15C20 22 22 20 22 15V9C22 4 20 2 15 2H9C4 2 2 4 2 9V15C2 20 4 22 9 22Z" stroke="#373D3F" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+                                    </svg>
+                                </button>
+                            {{/if}}
+                        </div>
+                        <div class="c-cart__item-total">
```

Comment:
could it be a separate component called `c-order-total` ? The same data is being displayed in order's details on account page so we could make the component more general